### ISSUE TITLE
Should omitNestedClosingTags be actually required?

### ISSUE THREAD

**Issue by deluksic** [solidjs/vite-plugin-solid#149]
https://github.com/solidjs/vite-plugin-solid/blob/dfdd6d2f6cbd0de2e7f69872275eae5c3a8b9066/src/index.ts#L87

Should it actually be required, or is this an accidental omission? (missing `?`)

**Comment by ryansolid** [comment]
Yeah this is odd. I've changed it. WIll be fixed in the next release.
